pages/search/[[term]].vue

```diff
@@ -10,6 +10,9 @@ import type {
 const router = useRouter();
 const { t } = useI18n();
 toolbarTitleStore().setReactiveToolbarTitle(() => "");
```

Comment:
This should probably we replaced with the `setTitle` function you introduced in other pages

Reply:
Ah, nevermind. I see that the `setTitle` function only should be used if the toolbar title and page title are the same